index.js

```diff
@@ -68,7 +76,35 @@ function toggleColor(element) {
   }
 }
 
+
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector("button")
+
+likeButton.addEventListener("click", function addLikes(event) {
+  const theLikes = document.querySelector(".likes") 
+  const addLikes = parseInt(theLikes.textContent) + 1
+  theLikes.textContent = `${addLikes} Likes`
+} )
+
 
 /***** Deliverable 3 *****/
+
+const goalForm = document.querySelector("#new-goal-form")
+
+goalForm.addEventListener("submit", function(event) {
+  event.preventDefault()
+  const link = event.target.link.value
+  const desc = event.target.description.value
+
+  const goalObj = {
+    link: link,
+    description: desc,
+    playerId: player.id,
+    id: 3
+
+  }
+
+  renderGoal(goalObj)
```

Comment:
Looks good! When you're making the `newGoal` variable, if you want to save on variables declarations, you could also create your object like so:

```js
  const newGoal = {
    link: event.target.link.value,
    description: event.target.description.value,
    playerId: player.id,
    id: 3 // when we learn about fetch, we'll use the server to generate this ID instead
  }
```